generated/nidcpower/nidcpower/_converters.py

```diff
@@ -275,3 +276,19 @@ def convert_chained_repeated_capability_to_parts(chained_repeated_capability):
     return [','.join(collections.OrderedDict.fromkeys(x)) for x in repeated_capability_lists]
 
 
+def convert_from_isolation_state_enum(enum):
+    if type(enum) is not enums._IsolationState:
```

Comment:
* TypeError doesn't seem like the right thing to raise. Probably ValueError. This would only happen if the driver runtime returns something unexpected, right?
* Ask for forgiveness, not for permission, in other words, do a try/except. Don't check first.

Reply:
> This would only happen if the driver runtime returns something unexpected, right?

Yes, you're right.

This explicit `type()` check was actually based on the [existing implementation of the setter method of `AttributeEnum` class](https://github.com/ni/nimi-python/blob/master/generated/nidcpower/nidcpower/_attributes.py#L98). Nevertheless, I have changed it to use try/except in my latest commit.